docs/source/reference/samplers/index.rst

```diff
@@ -20,13 +20,13 @@ The :mod:`~optuna.samplers` module defines a base class for parameter sampling a
 +----------------------------------+---------------+---------------+----------------------+----------------+----------------------------------------+---------------+----------------+
 | Categorical parameters           |       ✅      |      ✅       |          ✅          |       ▲        |                   ✅                   |       ▲       |       ✅       |
 +----------------------------------+---------------+---------------+----------------------+----------------+----------------------------------------+---------------+----------------+
-| Pruning                          |       ✅      |      ✅       |          ✅          |       ▲        |                   ❌                   |       ✅      |       ▲        |
+| Pruning                          |       ✅      |      ✅       |          ✅          |       ▲        |                   ❌                   |       ✅      |       ❌       |
```

Comment:
I think ▲ is OK because the definition is "Works, but inefficiently."

Reply:
I agree with @not522.
I wonder if RandomSampler, GridSampler, and QMCSampler should probably be ▲ from the perspective you mentioned...?
>  does not have any interface to use information of pruned trials.

Reply:
I think `RandomSampler`, `GridSampler`, and `QMCSampler` should be ✅ because pruning can accelerate the optimization. For smarter samplers (I mean using trials' information), we judge them by using the information of pruned trials or not.
In my understanding, ✅ represents "the developers recommend this feature." Then, the current table looks good to me.

Reply:
Now I see the difference between [RandomSampler, GridSampler, QMCSampler] and [CmaEsSampler, BoTorchSampler]. It makes sense that ✅ represents "the developers recommend this feature".
Thanks for elaborating!